internal/versions/versions.go

```diff
@@ -175,42 +175,103 @@ func (h *handler) GetDefaultReleaseImage(cpuArchitecture string) (*models.Releas
 
 // Returns the OsImage entity
 func (h *handler) GetOsImage(openshiftVersion, cpuArchitecture string) (*models.OsImage, error) {
+	if cpuArchitecture == "" {
+		// Empty implies default CPU architecture
+		cpuArchitecture = common.DefaultCPUArchitecture
+	}
+	// Filter OS images by specified CPU architecture
+	osImages := funk.Filter(h.osImages, func(osImage *models.OsImage) bool {
+		if swag.StringValue(osImage.CPUArchitecture) == "" {
```

Comment:
This is strange, I think it's safe to assume that we've configured the service correctly, right?

Reply:
That's fine, this can wait for a proper refactoring.